src/main/java/be/acara/events/service/EventServiceImpl.java

```diff
@@ -65,6 +65,11 @@ public Event addEvent(Event event) {
         if (event.getId() != null) {
             throw new IdAlreadyExistsException("A new entity cannot already contain an id");
         }
+
+        String youtubeUrl = event.getYoutubeUrl();
+        String[] youtubeArrayUrl = youtubeUrl.split("=");
```

Comment:
Why not use the unused front end method here?

Reply:
It was a method I found on the internet. Forgot to delete it when I pushed.